src/components/inputs/CopyInput/examples/CopyInputExample.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { CopyInput } from "../CopyInput";
+
+export const CopyInputExample = () => {
+	const [invalid, setInvalid] = React.useState(false);
+	const onChange = (newValue: string) => {
+        setInvalid(newValue.length < 5 ? true : false);
+	};
+        const offset: [number, number] = [0 , 15];
```

Comment:
There's something weird with the formatting here... maybe a mix of tabs & spaces?